src/frontend/components/NetworkStatus/index.tsx

```diff
@@ -1,5 +1,137 @@
-import { Card } from "@stellar/design-system";
+import { useState, useEffect } from "react";
+import { Card, TextLink, Icon, Tag } from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+import moment from "moment";
```

Comment:
The Moment dev's generally advocate for not using the lib anymore (esp in new projects). 
https://momentjs.com/docs/

It might be best to not use it in new v2 code and start coming up with a plan to refactor old instances of it